### PR TITLE
HOV-104 | Create Util function to handle custom values

### PR DESCRIPTION
**Pull request by Raalzz**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: packages/lib/src/components/Radio/radio.styles.css.ts [hunk 1]
```diff
@@ -23,6 +23,7 @@ export const radioSizes: Record<TRadioTheme[1]["radioStyleSize"], string> = {
   xl: "48px"
 };
 
+//Radio Default Values
```
Comment (pruthvi2103) [inline packages/lib/src/components/Radio/radio.styles.css.ts:26]: remove comment

**Review by pruthvi2103 — CHANGES_REQUESTED** [review]
Just remove that comment

File: packages/lib/src/components/Radio/radio.styles.css.ts (hunk 1 above)
Comment (Raalzz) [reply packages/lib/src/components/Radio/radio.styles.css.ts]: done

**Review by pruthvi2103 — APPROVED** [review]
LGTM

**Pull request merged**